spec/concepts/api/v1/lib/operation/filtering_spec.rb

```diff
@@ -142,29 +142,29 @@
     context 'when invalid filterable column' do
       let(:available_columns) { create_available_columns(name: 'attribute_1', filterable: false) }
 
-      let(:errors) { { filter: [[0, [I18n.t('errors.filtering_column_valid?')]]] } }
+      let(:errors) { { 0 => [I18n.t('dry_validation.errors.filtering_column_valid?')] } }
```

Comment:
[Rule](https://github.com/rubygarage/boilerplate/pull/222/files#diff-920c07dab8fe4fa10044c3508414c98395b314e2d92f32553b811ebab6e974c2R14) is match to [documentation](https://dry-rb.org/gems/dry-validation/1.5/rules/)
But for some reason it terurns just last key with message errors.
When we call contract from rails console we will get correct keys inheritance with `filter` key.
Could some one explaine it?